src/pages/OnboardEngagement/PurposeForUsingExpensifyPage.tsx

```diff
@@ -23,44 +23,40 @@ import SCREENS from '@src/SCREENS';
 // This is not translated because it is a message coming from concierge, which only supports english
 const messageCopy = {
     [CONST.INTRO_CHOICES.TRACK]:
-        'Great! To track your expenses, I suggest you create a workspace to keep everything contained:\n' +
+        'Here\'s how to start tracking business spend for taxes:\n' +
```

Comment:
Looks like our style linter prefers using `"` over `'` in the cases where we have apostrophes in the copy 
```suggestion
        "Here's how to start tracking business spend for taxes:\n" +
```